apps/user_profile/forms.py

```diff
@@ -0,0 +1,119 @@
+from django.forms import CharField, Form, PasswordInput, IntegerField
+from django import forms
+from django.views.generic import FormView
+
+from apps.user_profile.models import Child
+
+
+class RegisterChildForm(forms.ModelForm):
+        class Meta:
+            model = Child
+            fields = ('first_name', 'last_name', 'surname', 'dob', 'sex', 'city', 'street', 'house', 'flat',
+                      'birth_sertificate_series', 'birth_sertificate_number', 'birth_sertificate',
+                      'privilege', 'privilege_series', 'privilege_number')
+
+
+    # last_name = forms.CharField( max_length=100)
+    # first_name = forms.CharField(max_length=100)
+    # surname = forms.CharField(max_length=100)
+    # gender = forms.CheckboxSelectMultiple()
+    # dob = forms.DateField()
+    # register_city = forms.CharField(max_length=100)
+    # register_street = forms.CharField(max_length=250)
+    # register_house = forms.CharField(max_length=100)
+    # register_appart_number = forms.IntegerField()
+    # doc_series = forms.CharField(max_length=20)
+    # doc_number = forms.IntegerField()
+    # doc_info = forms.CharField(max_length=250)
+    # previlege = forms.CheckboxSelectMultiple()
+    # previlege_series= forms.CharField(max_length=100)
+    # previlege_number = forms.IntegerField()
+
+
+'''GENDER_CHOICES = (
+    ('жіноча', "Жіноча"),
+    ("чоловіча", "Чоловіча")
+)
+
+PRIVILEGE_CHOICES = (
+    (0, "---"),
+    (1, 'Інваліди I-шої групи'),
+    (2, 'Діти-інваліди, які не мають протипоказань до відвідування ДНЗ'),
+    (3, 'Діти Військовослужбовців Збройних сил України, Національної гвардії учасників АТО'),
+    (4, 'Учасник ліквідації наслідків катастрофи на Чорнобильській АЕС, потерпілих від екологічного та іншого стихійного лиха, аварій, катастроф'),
+    (5, 'Діти під опікою')
+)
+
+
+class RegisterChildView(Form):
+
+    lastname = CharField(
+        label='Прізвище дитини',
+        max_length=100,)
+    name = CharField(
+        label='Ім\'я дитини' ,
```

Comment:
What did I see? `\`
if you use " " (double quotes) then you do not need shield the apostrophe